livre/eq/settings/RenderSettings.h

```diff
@@ -1,7 +1,7 @@
 
 /* Copyright (c) 2011-2016, Maxim Makhinya <maxmah@gmail.com>
  *                          David Steiner  <steiner@ifi.uzh.ch>
- *                          Ahmet Bilgili  <ahmet.bilgili@epfl.ch>
+ *                          Stefan.Eilemann@epfl.ch
```

Comment:
Euh... it's good you spotted that one... I have no idea what happened... Most probably a copy paste shit during the rebase